src/Community/data.json

```
@@ -0,0 +1,78 @@
+{
+  "hero": {
+    "picture": "/static/img/community/hero.png",
+    "url": "http://event.link"
+  },
+  "section": {
+    "contribute": {
+      "title": "Contribute",
+      "description": "DVC is 100% open-source and we welcome contributions of all sorts to our code and docs."
```

```suggestion
      "description": "DVC is 100% open source. We welcome contributions to our code and docs."
```
per https://github.com/iterative/dvc.org/pull/874#issuecomment-571841464

---

p.s. @iAdramelk is the pre-commit hook not asking you to rewrap these lines? I see Restyled has some complaints BTW (but not this one). Are you using `git commit --no-verify`?

---

Nope. I think the problem is that restyled didn't read `.prettierrc` and is using it's own defaults to validate. I think that this is fixed in master, but branch is behind this change.

---

Ah OK. Yeah so feel free to merge from `master` any time. Thanks!

---

Suggestion still pending 🙂